src/codeu/chat/client/ClientConversation.java

```diff
@@ -41,6 +45,8 @@
 
   private final ClientUser userContext;
   private ClientMessage messageContext = null;
+  
+  private final Map<Uuid, Conversation> conversationsByUuid = new HashMap<>();
```

Comment:
And while I'm being annoying, I'll throw in the idea of scalability into the mix: how large could this `Map` get? What if you had a million users with a million conversations each? How would you change your code to handle that case?